forge/src/multi_runner.rs

```diff
@@ -548,6 +570,22 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_fork() {
```

Comment:
This test should probably be skipped if `ETH_RPC_URL` is not set, similar to https://github.com/gakonst/foundry/blob/90617a52e4873f0137aa05fd68624437db146b3f/cli/test-utils/src/macros.rs#L134-L137